packages/@lwc/integration-karma/scripts/karma-configs/hydration/base.js

```diff
@@ -9,6 +9,7 @@
 
 const path = require('path');
 
+const { globSync } = require('fs');
```

Comment:
```suggestion
const { globSync } = require('node:fs');
```

We're not avoiding `node:*` intentionally; we just haven't fixed every missing `node:` prefix. Same for the `path` above – feel free to clean up as you go. 🙂

Reply:
Changed that one and the path require too.